webpack.dev.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path')
+const webpack = require('webpack')
+
+module.exports = {
+  entry: './client.jsx',
+  mode: 'development',
+  watch: true,
+  module: {
+    rules: [{
+      test: /\.(js|jsx)$/,
+      exclude: /node_modules/,
+      use: ['babel-loader'],
+    }],
+  },
+  resolve: {
+    extensions: ['.js', '.jsx'],
+  },
+  output: {
+    path: path.resolve(__dirname, 'public', 'javascript'),
+    publicPath: '/',
+    filename: 'bundle.js',
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      API_BASE_URL: JSON.stringify('http://localhost:1337/api'),
```

Comment:
This doesn't match your server, your server used 1447. This causes your app to break

![image](https://user-images.githubusercontent.com/5775716/85047945-8c79f100-b160-11ea-952b-b6d5750f875e.png)